.storybook/stories/Introduction.stories.mdx

```diff
@@ -21,3 +21,41 @@ The design tokens are [available as CSS variables on GitHub](https://github.com/
 ## Design Kit for Figma
 
 Find our design kit for [on Figma](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/).
+
+## Getting started
+
+This getting started section explains how to use the React Components in your project.
+
+### Add required dependencies
+
+Add the NPM package for the components. The component library can be imported as a whole, or per component.
+
+In order to add the entire component library, install [@gemeente-denhaag/denhaag-component-library](https://www.npmjs.com/package/@gemeente-denhaag/denhaag-component-library) using your favorite package manager.
+
+For example, ```npm install @gemeente-denhaag/denhaag-component-library```.
+
+Alternatively, you can install specific components: ```npm install @gemeente-denhaag/button```.
+
+After installing the dependency, you can start using the Den Haag components as follows:
+```jsx
+import {Button, StylesProvider} from "@gemeente-denhaag/denhaag-component-library";
+// Import your design tokens here as well! Otherwise your components will be unstyled :(
+
+function App() {
+    return (
+        <StylesProvider>
+            <Button variant={"primary-action"}>Click here!</Button>
+        </StylesProvider>
+    );
+}
+
+export default App;
+```
+**Important:** Your app, or the section which uses Den Haag components, will need to be wrapped in the `StylesProvider` component. This ensures that styles are loaded correctly.
+
+#### Applying your theme
+
+By default, all components in the library are unstyled.
+If you wish to apply your organisations theme to the components, define the design tokens for the components you need, and import them in your app.
+
+For Den Haag applications, import the [@gemeente-denhaag/design-tokens-components](https://www.npmjs.com/package/@gemeente-denhaag/design-tokens-components) package, and import the css in your application.
```

Comment:
Maybe also add an example here of how to import these?

Reply:
Added examples for js import as well as regular html head import